osgar/tools/log2pcap.py

```diff
@@ -56,5 +56,9 @@ def log2pcap(input_filepath, output_dir):
     for filename in sys.argv[1:-1]:
         log2pcap(filename, output_dir)
 
+
+if __name__ == "__main__":
```

Comment:
I have not noticed this changes to `def main()` ... what is the motivation? (BTW it will cause a small conflict with other branches with lidarview)